File: 01-Ejercicios-Clases/C4-08-25-23/Ejercicio02/Persona.cs
class Persona {
	private string nombre;
	//Constructo
	public Persona(string nombre){
		this.nombre = nombre;
	}
	//get y set del atributo nombre
	public string Nombre{
		get{
			return nombre;
		}
		set{
			this.nombre = value;
		}
	}
}

File: 01-Ejercicios-Clases/C2-08-18-23/Persona.java
abstract class Persona implements IComun{
	private String nombre;
	String apellidoPaterno;
	static float CRAEST;
	public Persona(){System.out.println("Mensaje Persona");}
	public Persona(String nombre){
		this.nombre = nombre;
	}
	public Persona(String nombre, String apellidoPaterno){
		this.nombre = nombre;
		this.apellidoPaterno = apellidoPaterno;
	}
	public void setNombre(String nombre){
		this.nombre = nombre;
	}
	public String getNombre(){
		return nombre;
	}
	
	public abstract void imprimirDatos();
	
	public abstract void mostrarCodigoPUCP();
	
	/*
	public void finalize(){
		System.out.println("El objeto esta siendo destruido");
	}*/
	
	public static void mostrarMensaje(){
		System.out.println("Hola mundo");
	}	
	
	
	
}